modules/multi-arch-installing-using-cli.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-installing-using-cli_{context}"]
+
+= Installing the Multiarch Tuning Operator by using the CLI
+
+You can install the Multiarch Tuning Operator by using the {oc-first}.
```

Comment:
You might add a very basic summary of why we create an OG object and Subscription object in this procedure.

Reply:
This is generic info. It is available in this section: https://docs.openshift.com/container-platform/4.15/operators/user/olm-installing-operators-in-namespace.html#olm-installing-operator-from-operatorhub-using-cli_olm-installing-operators-in-namespace

In the context of this Operator, I think adding details about OG object and Subscription object is not needed.